backend/database/models/order.model.ts

```diff
@@ -0,0 +1,46 @@
+import { getModelForClass, modelOptions, prop, Ref } from '@typegoose/typegoose'
+import { User } from './user.model'
+
+@modelOptions({ options: { allowMixed: 0 } })
```

Comment:
What's this `allowMixed` used for?

Reply:
set to solve the "setting the mixed property" problem